mkdocs.yml

```diff
@@ -67,20 +67,13 @@ nav:
       - metrics/accuracy.md
       - metrics/average-precision.md
       - metrics/averaging-methods.md
-      - metrics/bertscore.md
-      - metrics/bleu.md
       - metrics/confusion-matrix.md
       - metrics/f1-score.md
-      - metrics/fpr.md
       - metrics/geometry-matching.md
       - metrics/iou.md
-      - metrics/meteor.md
       - metrics/precision.md
       - metrics/pr-curve.md
       - metrics/recall.md
-      - metrics/roc-curve.md
-      - metrics/rouge-n.md
-      - metrics/specificity.md
```

Comment:
Is this necessary to revert?

Reply:
yeah there are some new pages like fpr and specificity that never had the original diagrams.